kubernetes/resources_api/base/deployment.yaml

```diff
@@ -0,0 +1,91 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: resources-api
+spec:
+  replicas: 2
+  revisionHistoryLimit: 5
+  template:
+    spec:
+      containers:
+      - name: app
+        image: #operationcode/operationcode_backend:latest
+        imagePullPolicy: Always
+        ports:
+        - containerPort: 3000
+        env:
+        - name: FLASK_APP
+          value: run.py
+        - name: SQLALCHEMY_DATABASE_URI
```

Comment:
Does the DB URI contain secrets? Generally it wouldn't, unless it's the entire connection string. If no secrets, doesn't need to come from a secret :)

Reply:
In the `env` file, we specify the full connection string with username and password. It might take a slight bit of refactoring to change that such that we don't need it... I'm not really sure to be honest. Is it fine if we leave the full connection string in there?